doc/API/Switching.md

```diff
@@ -277,3 +277,96 @@ Output:
     'status': 'ok'
 }
 ```
+
+### `list_nac`
+
+Get a list of all ports NAC.
+
+Route: `/api/v0/resources/nac/:mac`
+
+- mac is the specific MAC address you would like to query
+
+Input:
+
+-
+
+Example:
+
+```curl
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/resources/nac
+curl -H 'X-Auth-Token: YOURAPITOKENHERE' https://librenms.org/api/v0/resources/nac/1aaa2bbb3ccc
+```
+
+Output:
+
+```json
+{
+    "status": "ok",
+    "ports_nac": [
+        {
+            "ports_nac_id": 1,
+            "auth_id": "0000000000000AAAABBBB8CC",
+            "device_id": 3,
+            "port_id": 6,
+            "domain": "data",
+            "username": "hostname.librenms.org",
+            "mac_address": "1aaa2bbb3ccc",
+            "ip_address": "192.0.2.2",
+            "host_mode": "singleHost",
+            "authz_status": "authorizationSuccess",
+            "authz_by": "Authentication Server",
+            "authc_status": "authcSuccess",
+            "method": "dot1x",
+            "timeout": "0",
+            "time_left": "0",
+            "vlan": 0,
+            "time_elapsed": null,
+            "created_at": "2025-05-08T08:55:06.000000Z",
+            "updated_at": "2025-05-08T08:55:06.000000Z",
+            "historical": 0
+        }
+    ]
+}
+```
+
+
+### `list_nac_detail`
```

Comment:
Not sure detail is the right name for this since it contains less information than the above endpoint.

Also, this just seems like a shortcut from making multiple api calls, I'm not sure it adds much.